package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "camp2-helpers",
+  "version": "0.1.0",
+  "description": "",
+  "main": "dist/index.js",
+  "repository": "git@github.com:fewlines-education/camp2-helpers.git",
+  "author": "Fewlines",
+  "license": "MIT",
+  "private": false,
+  "devDependencies": {
+    "@types/jest": "^25.1.4",
+    "@typescript-eslint/eslint-plugin": "^2.24.0",
+    "@typescript-eslint/parser": "^2.24.0",
+    "eslint": "^6.8.0",
+    "eslint-config-prettier": "^6.11.0",
+    "eslint-plugin-prettier": "^3.1.3",
+    "jest": "^25.1.0",
+    "prettier": "^2.0.5",
+    "ts-jest": "^25.2.1",
+    "ts-node": "^8.7.0",
+    "typescript": "^3.8.3"
+  },
+  "eslintConfig": {
+    "parser": "@typescript-eslint/parser",
+    "parserOptions": {
+      "ecmaVersion": 6,
+      "sourceType": "module"
+    },
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended",
+      "prettier/@typescript-eslint",
+      "plugin:prettier/recommended"
+    ],
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "env": {
+      "browser": true,
+      "node": true
+    },
+    "ignorePatterns": [
+      "jest.config.js",
+      "node_modules/",
+      "*.test.ts",
+      "src.sample/*",
+      "build/"
+    ],
+    "rules": {
+      "@typescript-eslint/explicit-function-return-type": 1,
+      "@typescript-eslint/no-explicit-any": 0
```

Comment:
Was this copied from an other repo, or do you need those rules ? (the second one I think you need it, the first one shouldn't be necessary, as you allow `any` and can type any return function as such).

Reply:
This was straight up copied blindly from the readline package... 😅